ArduPlane/quadplane.cpp

```diff
@@ -4350,11 +4371,16 @@ bool SLT_Transition::allow_update_throttle_mix() const
     return !(quadplane.assisted_flight && (transition_state == TRANSITION_AIRSPEED_WAIT || transition_state == TRANSITION_TIMER));
 }
 
-bool SLT_Transition::active() const
+bool SLT_Transition::active_frwd() const
 {
     return quadplane.assisted_flight && ((transition_state == TRANSITION_AIRSPEED_WAIT) || (transition_state == TRANSITION_TIMER));
 }
 
+bool SLT_Transition::active_back() const
+{
+    return false;
```

Comment:
there is a back transition :-) 
we can remove active_back() for now to avoid having to define it

Reply:
I propose removing it and I have a commit ready for it.
See my comment on the other conversation.